src/crimson/os/seastore/transaction_manager.h

```diff
@@ -179,7 +179,14 @@ class TransactionManager : public ExtentCallbackInterface {
     auto v = pin->get_logical_extent(t);
```

Comment:
In addition to @aisakaki 's comments, looks if `v.has_child()` is true, it will always go to `Cache::get_extent_viewable_by_trans()`, which already makes sure that the returned extent is always fully loaded, synchronously.

Am I missing anything? Or how is the bug really happens?

Reply:
Seems it is completely caused by `ObjectDataBlock` not having a constructor for creating a not-fully-loaded extent. I'll remove the other commit.